### PR TITLE
3 create bar graph

### PR DESCRIPTION
**Pull request by cpshan**

### CODE REVIEW

**Review by mbrewer05 — APPROVED** [review]

File: frontend/src/components/BarChart.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+import { Bar } from "react-chartjs-2";
+import { MDBContainer } from "mdbreact";
+import axios from "axios";
+import {red, orange, yellow, green, blue, purple, deepPurple, 
+    pink, lightBlue, cyan, teal, lightGreen} from '@material-ui/core/colors';
+import IconButton from '@material-ui/core/IconButton';
+import KeyboardArrowRightIcon from '@material-ui/icons/KeyboardArrowRight';
+import KeyboardArrowLeftIcon from '@material-ui/icons/KeyboardArrowLeft';
+
+const userID = "60a483f4cc4a814ce0cb4139";
```
Comment (ghuarng) [inline frontend/src/components/BarChart.js]: Please make this compatible with multiple users (see Max)

File: frontend/src/components/BarChart.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+            May: 0,
+            June: 0,
+            July: 0,
+            August: 0,
+            September: 0,
+            October: 0,
+            November: 0,
+            December: 0
+        }
+        return (axios.get('http://localhost:5000/users/' + userID + '/transactions?spent=1')
```
Comment (ghuarng) [inline frontend/src/components/BarChart.js]: (see above)

**Review by ghuarng — CHANGES_REQUESTED** [review]
Please make compatible with multiple users (see Max)

**Review by ghost — APPROVED** [review]
lgtm

**Review by ghuarng — APPROVED** [review]
LGTM

**Pull request merged**